cartography/intel/aws/ses.py

```diff
@@ -0,0 +1,133 @@
+    MERGE (identity:AWSSESIdentity{id: record.arn})
+    ON CREATE SET identity.firstseen = timestamp(),
+        identity.arn = record.arn
+    SET identity.lastupdated = {aws_update_tag},
+        identity.name = record.name,
+        identity.region = record.region,
+        identity.dkim_enabled = record.dkim.DkimEnabled,
+        identity.dkim_verification_status = record.dkim.DkimVerificationStatus,
+        identity.verification_status = record.verification.VerificationStatus
+    WITH identity
+    MATCH (owner:AWSAccount{id: {AWS_ACCOUNT_ID}})
+    MERGE (owner)-[r:RESOURCE]->(identity)
+    ON CREATE SET r.firstseen = timestamp()
+    SET r.lastupdated = {aws_update_tag}
+    """
+
+    tx.run(
+        query,
+        Records=identities,
+        AWS_ACCOUNT_ID=current_aws_account_id,
+        aws_update_tag=aws_update_tag,
+    )
+
+
+@timeit
+def cleanup_ses_identities(neo4j_session: neo4j.Session, common_job_parameters: Dict) -> None:
+    run_cleanup_job('aws_import_ses_identity_cleanup.json', neo4j_session, common_job_parameters)
+
+
+@timeit
+def sync(
+    neo4j_session: neo4j.Session, boto3_session: boto3.session.Session, regions: List[str], current_aws_account_id: str,
+    update_tag: int, common_job_parameters: Dict,
+
+
+) -> None:
+    tic = time.perf_counter()
+
+    logger.info("Syncing SES for account '%s', at %s.", current_aws_account_id, tic)
+
+    identities = []
+    for region in regions:
+        logger.info("Syncing SES for region '%s' in account '%s'.", region, current_aws_account_id)
+
+        identities.extend(get_ses_identity(boto3_session, region, current_aws_account_id))
+
+    if common_job_parameters.get('pagination', {}).get('ses', None):
+        pageNo = common_job_parameters.get("pagination", {}).get("ses", None)["pageNo"]
+        pageSize = common_job_parameters.get("pagination", {}).get("ses", None)["pageSize"]
+        totalPages = len(identities) / pageSize
+        if int(totalPages) != totalPages:
+            totalPages = totalPages + 1
+
+        totalPages = int(totalPages)
+        if pageNo < totalPages or pageNo == totalPages:
+            logger.info(f'pages process for SES identities {pageNo}/{totalPages} pageSize is {pageSize}')
+
+        page_start = (common_job_parameters.get('pagination', {}).get('ses', {})[
+            'pageNo'] - 1) * common_job_parameters.get('pagination', {}).get('ses', {})['pageSize']
+        page_end = page_start + common_job_parameters.get('pagination', {}).get('ses', {})['pageSize']
+        if page_end > len(identities) or page_end == len(identities):
+            identities = identities[page_start:]
+
+        else:
+            has_next_page = True
+            identities = identities[page_start:page_end]
+            common_job_parameters['pagination']['ses']['hasNextPage'] = has_next_page
+
+    load_ses_identity(neo4j_session, identities, current_aws_account_id, update_tag)
+    cleanup_ses_identities(neo4j_session, common_job_parameters)
```

Comment:
last log statement is missing where we print the total time taken by this service.